MEEW-Client/MEEW-Client/Screens/ToDo/Controller/ToDoVC.swift

```diff
@@ -8,143 +8,124 @@
 import UIKit
 
 class ToDoVC: BaseVC {
-    
-    // MARK: - var & let
-    
+  
+  // MARK: - @IBOutlet
+  @IBOutlet weak var questionLabel: UILabel!
+  @IBOutlet weak var originalButton: UIButton!
+  @IBOutlet weak var archiveButton: UIButton!
+  @IBOutlet weak var inforButton: UIButton!
+  @IBOutlet weak var doneButton: UIButton!
+  @IBOutlet weak var imageView: UIImageView!
+  @IBOutlet weak var checkBoxButton1: UIButton!
+  @IBOutlet weak var checkBoxButton2: UIButton!
+  @IBOutlet weak var checkBoxButton3: UIButton!
+  @IBOutlet weak var checkBoxButton4: UIButton!
```

Comment:
다음과 같은 코드는 IBOutlet Collection을 이용해서 좀 더 깔끔하게 코드를 작성할 수 있을 것 같습니다.

```suggestion
  @IBOutlet weak var checkBoxButtons: [UIButton]!
```

간단하게 생각하면 우리가 일반적으로 같은 자료형들끼리 묶어서 하나의 배열로 사용하듯이
IBOutlet 역시 같은 성격의 요소를 배열 처럼 묶어서 사용할 수 있답니다. (이것이 IBOutlet 컬렉션)

주의할 점은 하나의 IBOutlet Collection 변수에 UIButton 컴포넌트를 모두 연결 지어주어야 한다는 점!

아래에 블로그 첨부할테니까 참고해주세요~
https://mini-min-dev.tistory.com/106

Reply:
아래의 라벨도 같은 방식으로 묶어줄 수 있겠죠?

Reply:
@Taehyeon-Kim 왁...대박..새로운거 알아갑니다..👍

Reply:
앆 깔끔을 좋아하는 사람으로서 정말 꿀팁입니다..바로 가서 수정해야겠어요!